magda-web-client/src/Components/Dataset/DatasetSummary.js

```diff
@@ -37,13 +38,26 @@ export default class DatasetSummary extends Component {
         const searchText = defined(this.props.searchText)
             ? this.props.searchText
             : "";
+        const searchResultNumber = this.props.searchResultNumber;
         return (
             <div className="dataset-summary">
                 <h2 className="dataset-summary-title">
                     <Link
                         to={`/dataset/${encodeURIComponent(
                             dataset.identifier
                         )}?q=${searchText}`}
+                        onClick={() => {
+                            if (searchResultNumber) {
```

Comment:
Had a test - if I watch chrome devtools I can see the other new events work, but not this one :(.

Reply:
<img width="943" alt="click search result" src="https://user-images.githubusercontent.com/81432/42788426-d093db54-89a2-11e8-83b0-62fcc54f9f7a.PNG">
It's not essential, happy to just try it out to see if it works consistently.